tests/bson-corpus/code_w_scope-decodeError-011.phpt

```diff
@@ -1,7 +1,5 @@
 --TEST--
 Javascript Code with Scope: bad scope doc (field has bad string length)
---XFAIL--
-Depends on PHPC-889
```

Comment:
Shouldn't this still fail until [PHPC-889](https://jira.mongodb.org/browse/PHPC-889) is resolved?